architecture/src/commonMain/kotlin/navigation/Navigator.kt

```diff
@@ -23,7 +23,21 @@ import com.splendo.kaluga.architecture.lifecycle.LifecycleSubscribable
  * Action that describes the intent to navigate
  * @param bundle The [NavigationBundle] containing data used to configure navigation
  */
-abstract class NavigationAction<B : NavigationBundleSpecRow<*>>(val bundle: NavigationBundle<B>?)
+open class NavigationAction<B : NavigationBundleSpecRow<*>>(val bundle: NavigationBundle<B>?)
+
+/**
+ * A [NavigationAction] that has a [SimpleNavigationSpec] bundle
+ * @param value The value passed by the action
+ * @param type The [NavigationBundleSpecType] describing the object passed by the action
+ */
+open class SimpleNavigationAction<T>(
```

Comment:
I'd name it `ObjectNavigationAction` or `SingleValueNavigationAction`, simple in name doesn't describe why it's simple